toolkit/tools/scheduler/scheduler.go

```diff
@@ -466,6 +463,53 @@ func updateGraphWithImplicitProvides(res *schedulerutils.BuildResult, pkgGraph *
 	return
 }
 
+// setAssociatedDeltaPaths sets the RpmPath for all of the request's ancillary nodes (both build and run) to the actual
+// RPM paths. A delta node will normally point at the cached RPM path, but we want to point it at the actual RPM if we built it.
+// This function should only be called on delta build nodes.
+func setAssociatedDeltaPaths(res *schedulerutils.BuildResult, builtFiles []string, pkgGraph *pkggraph.PkgGraph, graphMutex *sync.RWMutex) (err error) {
```

Comment:
>res *schedulerutils.BuildResult [](http://example.com/codeflow?start=29&length=31)

I think `res` is unused.

Reply:
SHould just have `res`, `builtFiles` is redundant since that is part of `res`.